src/config.rs

```diff
@@ -74,6 +103,7 @@ pub struct Config {
     #[serde(skip)]
     pub dump: Vec<Dump>,
     pub syntax_highlight: SyntaxHighlight,
+    pub sass: SassOptions,
```

Comment:
Have you rebased against master?  I don't think this is going to work quite as you expect.

Like with frontmatter, I've decoupled the external and internal implementations of `Config`.  You've added support for sass to the internal implementation but not the external (what we actually use to read the file).

Why did I add this extra layer of more work?

This is to open  the door for us to experiment with evolving the config format while deferring breaking compatibility.  Particularly, I want to create a conversion utility when we do break compatibility to lessen the pain.  I believe this work is being tracked in #199.

Reply:
> Have you rebased against master?
Not since I started no.

>  I don't think this is going to work quite as you expect.

okay..... Well it did work for the test case I had added. But I suppose I should rebase against master and see for myself, as I don't quite understand what else you referring to. Have you seen the test case and would you expect it to fail? If not, could we design a test case that will fail now 
without the "frontmatter" implementation?

Reply:
I expect the test with custom settings to fail.  Either it will report an error about unsupported fields (ideally) or it will act as if those fields weren't set.

The relevant [lines of code](https://github.com/cobalt-org/cobalt.rs/blob/master/src/config.rs#L118):
```rust
let config: wildwest::GlobalConfig = serde_yaml::from_str(&content)?;
let mut config: Config = config.into();
```

[`wildwest::GlobalConfig`](https://github.com/cobalt-org/cobalt.rs/blob/master/src/legacy/wildwest.rs#L83) represents the current `.cobalt.yml` file format.

[`config::Config`](https://github.com/cobalt-org/cobalt.rs/blob/master/src/config.rs#L54) represents what will eventually become the `.cobalt.yml` file format.

Reply:
I rebased and Added the wildwest config, but I am still unclear wether I understood you right. Do you mind taking a look?

Reply:
Looks great!